### DISCUSSION TITLE
Documentation netsuite laravel

### DISCUSSION THREAD

**Discussion by Ferdinandgushcloud** [netsuitephp/netsuite-php#125] [Q&A] [answered]
Hi Support. Currently, i want to integrate netsuite on my laravel project. I still new with netsuite. Is there any full documentation for integrate this netsuite laravel. There are so many Netsuite class that i don't really understand to use it. 

Please help

**Comment by jrebs** [comment, marked as answer]
Integrating this library into a laravel project is simply a matter of `composer require ryanwinchester/netsuite-php` and then using the library as you would any other in your laravel projects. As a convenience, there's a [netsuite-laravel](https://github.com/netsuitephp/netsuite-laravel) package which makes it a bit easier to pull the lib into a laravel project by adding the ability to access the NetSuite object via the service container or a Facade. The netsuite-laravel wrapper has its own documentation on how to pull it in and use it.

There's nothing that is going to eliminate the fact that the NetSuite web services system depends on over a thousand classes. This is just the way the web services is designed. There are links to the NetSuite documentation pages for web services at the [EXAMPLES](https://github.com/netsuitephp/netsuite-php/blob/master/EXAMPLES.md#resources) file.
